portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase

```diff
@@ -0,0 +1,54 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Check for errors in the minified browser console"
+	@priority = "3"
+	test SmokeMinifiedEnabled {
+		property custom.properties = "minified.enabled=true";
+
+		task ("Restart portal") {
+			Portlet.shutdownServer();
+
+			Portlet.startServer(deleteLiferayHome = "false");
+		}
+
+		task ("login") {
+			User.firstLoginPG();
+		}
+
+		task ("No liferay log or js console log errors") {
+			User.viewLoggedInPG();
+		}
+
+		task ("Smoke test passes") {
+			Smoke.runSmoke();
+		}
```

Comment:
Adjust the tasks to include the `Given When Then` terms from the test scenario.

```
Given: minified property set to true
When: restart portal
And When: Login
Then: No liferay log or js console log errors
And Then: Smoke test passes
```
see https://github.com/mateusralv/liferay-portal/pull/41/files#diff-3fdc3c718f6e440ff2b4cee5b74080c0c3ecc97076cd214ea19df03cfdd9f16dR101-R118

Reply:
Modification completed